app/views/form/_check_answers_summary_list.html.erb

```diff
@@ -28,7 +28,7 @@
       <% if @log.collection_period_open_for_editing? %>
         <% row.with_action(
           text: question.action_text(@log),
-          href: action_href(@log, question.page.id, referrer),
+          href: check_answers_href(question, @log),
```

Comment:
I think "referrer" isn't always "check_answers", I think it might be "interruption_screen" or _maybe_ something duplicate log related too. Is that addressed somewhere?

Reply:
Good point - I think I broke this while trying to tidy it up 🤦